### PR TITLE
[ID] Update all articles from the `Rules` category

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
Song content rules and Explicit content articles will follow #6048 edit as soon as it finalized.

### CODE REVIEW

File: wiki/Rules/Explicit_Content/id.md [hunk 1]
```diff
@@ -17,7 +12,7 @@ osu! mengizinkan pemakaian **konten audio** yang bersifat eksplisit (baik dari s
 
 **Di sisi lain, osu! tidak membenarkan penggunaan gambar maupun video yang bersifat senonoh - seluruh elemen visual yang ada tetap harus tunduk pada [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations).**
 
-Pengecualian di sini hanya diberikan pada elemen-elemen visual senonoh yang mengandung unsur estetika seni (*artistic use*) dalam nuansa dan konteks pemakaiannya.
+Pengecualian di sini hanya diberikan pada elemen-elemen visual senonoh yang mengandung unsur estetika seni (*artistic use*) dalam nuansa dan konteks pemakaiannya, seperti [storyboards](/wiki/Storyboards) atau konten video.
```
Comment (Nivalyx) [inline wiki/Rules/Explicit_Content/id.md:15]: `storyboards` should be `storyboard` as the Indonesian language doesn't use `s` at the end of a word to indicate a plural form

File: wiki/Rules/Song_Content_Rules/id.md [hunk 2]
```diff
@@ -1,32 +1,12 @@
...
 # Aturan konten musik
 
 Dengan banyaknya variasi musik yang tersedia, sebagian besar lagu akan baik-baik saja untuk digunakan di osu!, asalkan tidak:
 
-- Sangat eksplisit, menjurus, atau sangat konfrontatif dan berhubungan dengan konten yang mengejutkan, cabul, atau sangat seksual
-- Mengangkat, mendukung, atau mendorong penyalahgunaan narkoba, bunuh diri, pembunuhan, atau melukai diri sendiri
-- Mendukung penekanan dan perpecahan terhadap rasial
-- Mengandung suara atau arti sangat keras dan tak tertahankan atau terlalu terpotong
 - Mengandung hal yang sangat politis
+- Mendukung tekanan dan perpecahan terhadap rasial
```
Comment (Nivalyx) [inline wiki/Rules/Song_Content_Rules/id.md:6]: uhh `racial tension` doesn't literally mean `perpecahan terhadap rasial` orz :(

imo you could try sth like

`Bersifat diskriminatif terhadap suatu ras ataupun kalangan tertentu`

File: wiki/Rules/Song_Content_Rules/id.md [hunk 3]
```diff
@@ -1,32 +1,12 @@
...
+- Mengandung volume lagu sangat keras dan tak tertahankan atau terlalu durasi lagu terpotong terlalu banyak
```
Comment (Nivalyx) [inline wiki/Rules/Song_Content_Rules/id.md:7]: yeah it's the same thing here -- in audio-related terms the word `clipped` here doesn't literally translate into `an audio file that's being cut` orz. instead it's referring to `an audio file that's excessively loud` (http://en.wikipedia.org/wiki/Clipping_(audio)) if any

--> `Memiliki volume lagu yang dengan sengaja dipasang sedemikian keras hingga memunculkan efek *clipping* yang berlebihan`

File: wiki/Rules/id.md [hunk 4]
```diff
@@ -46,7 +41,7 @@ Peraturan-peraturan berikut ini merupakan pedoman dasar yang kami terapkan di da
...
-2. **Jangan membahas hal-hal yang tidak perlu untuk dibahas pada laman diskusi beatmap.** Laman diskusi beatmap dikhususkan untuk (dan hanya untuk) membahas hal-hal yang berkaitan dengan beatmap yang bersangkutan. Apabila kamu merasa perlu untuk membahas sesuatu yang tidak berhubungan dengan beatmap yang bersangkutan, kamu dapat membuka topik bahasan baru pada forum yang lebih sesuai.
+2. **Jangan membahas hal-hal yang tidak perlu untuk dibahas pada laman [diskusi beatmap](/wiki/Beatmap_Discussions).** Laman diskusi beatmap dikhususkan untuk (dan hanya untuk) membahas hal-hal yang berkaitan dengan beatmap yang bersangkutan. Apabila kamu merasa perlu untuk membahas sesuatu yang tidak berhubungan dengan beatmap yang bersangkutan, kamu dapat membuka topik bahasan baru pada forum yang lebih sesuai.
```
Comment (Nivalyx) [inline wiki/Rules/id.md:44]: `laman [diskusi beatmap](/wiki/Beatmap_Discussions)` -> `[laman diskusi beatmap](/wiki/Beatmap_Discussions)`

File: wiki/Rules/Song_Content_Rules/id.md (hunk 2 above)
Comment (ZeroPyrozen) [reply wiki/Rules/Song_Content_Rules/id.md:6]: Better wording and translation 👍

File: wiki/Rules/Song_Content_Rules/id.md (hunk 3 above)
Comment (ZeroPyrozen) [reply wiki/Rules/Song_Content_Rules/id.md:7]: Ah I thought clipping is the same as trimming

**Comment by ZeroPyrozen** [comment]
I'm going to check `osu!team` word in all articles and change it into `tim inti osu!` to fit #6086

File: wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md [hunk 5]
```diff
@@ -1,20 +1,15 @@
...
+**Kode Etik dalam Modding dan Mapping** adalah kumpulan dari beberapa aturan dan pedoman yang berlaku untuk seluruh aktivitas [modding](/wiki/Modding) dan [mapping](/wiki/Beatmapping) di dalam osu!, pedoman ini juga merupakan tambahan untuk [aturan komunitas osu!](/wiki/Rules) dan [Kriteria Ranking Umum](/wiki/Ranking_Criteria). Untuk memastikan diskusi modding dan mapping tetap berlangsung terkendali, konstruktif, positif dan produktif, kode etik perannya sangat penting untuk membuat semua orang tetap berada dalam topik pembahasan. Saat berpartisipasi dalam aktivitas modding dan mapping, kamu wajib mengikuti Kode Etik ini. [Sanksi akan dijatuhkan pada akunmu jika terbukti melakukan pelanggaran terhadap pedoman ini](/wiki/Silence).
```
Comment (mousesalt) [inline wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md:3]:
```suggestion
**Kode Etik dalam Modding dan Mapping** adalah kumpulan dari beberapa aturan dan pedoman yang berlaku untuk seluruh aktivitas [modding](/wiki/Modding) dan [mapping](/wiki/Beatmapping) di dalam osu!, pedoman ini juga merupakan tambahan untuk [aturan komunitas osu!](/wiki/Rules) dan [Ranking Criteria](/wiki/Ranking_Criteria). Kode etik memiliki peran penting untuk membuat semua orang tetap berada dalam topik pembahasan, memastikan diskusi modding dan mapping tetap berlangsung terkendali, konstruktif, positif dan produktif. Saat berpartisipasi dalam aktivitas modding dan mapping, kamu wajib mengikuti kode etik ini. [Sanksi akan dijatuhkan pada akunmu jika terbukti melakukan pelanggaran terhadap pedoman ini](/wiki/Silence).
```

Comment (mousesalt) [reply wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md:3]: referring to wording transition the sentences might be easier to understand.

File: wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md [hunk 6]
```diff
@@ -1,20 +1,15 @@
...
 
 ## Perilaku dan Etika
 
-- Tujuan dari modding adalah untuk membantu mapper meningkatkan kualitas beatmap yang mereka kerjakan supaya dapat diajukan statusnya menjadi Ranked jika memang tujuan mapper-nya ke sana. Kalau kamu tidak tertarik memberi umpan balik, hindari percakapan dan diskusi yang mungkin dapat menyakiti *perasaan mereka*.
+- Tujuan dari modding adalah untuk membantu mapper meningkatkan kualitas beatmap yang mereka kerjakan supaya dapat diajukan statusnya menjadi [Ranked](/wiki/Beatmap/Category#ranked) jika memang tujuan mapper-nya ke sana. Kalau kamu tidak tertarik memberi umpan balik, hindari percakapan dan diskusi yang mungkin dapat menyakiti *perasaan mereka*.
```
Comment (mousesalt) [inline wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md:7]:
```suggestion
- Tujuan dari modding adalah untuk membantu mapper meningkatkan kualitas beatmap yang mereka kerjakan supaya dapat diajukan menjadi status [ranked](/wiki/Beatmap/Category#ranked), jika memang tujuan mapper-nya ke sana. Kalau kamu tidak tertarik memberi umpan balik, hindari percakapan dan diskusi yang mungkin dapat menyakiti *perasaan mereka*.
```

File: wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md [hunk 7]
```diff
@@ -52,14 +47,14 @@ Seseorang baru saja memposting beberapa saran di beatmap milikmu, kerja bagus! -
...
 
 - Jika kamu belum yakin mengenai suatu hal, minta orang tersebut untuk menjelaskan apa maksudnya. Mereka harus lebih dari siap untuk menjelaskan berbagai hal kepadamu lebih rinci.
-- Masih belum bisa mencapai kesepakatan? Minta opini kepada seseorang yang memiliki lebih banyak pengalaman, seperti anggota BN atau NAT. Tetapi ingatlah, jadwal mereka padat! Sadarlah akan banyaknya alternatif yang lebih cocok dibanding sesuatu yang kamu sarankan sebelumnya, jangan takut untuk mengesampingkan ego dirimu - **terkadang melakukan kesalahan bisa jauh lebih berharga ilmu dan pengalamannya dibanding menjadi benar setiap saat**.
+- Masih belum bisa mencapai kesepakatan? Minta opini kepada seseorang yang memiliki lebih banyak pengalaman, seperti anggota [BN](/wiki/People/The_Team/Beatmap_Nominators) atau [NAT](/wiki/People/The_Team/Nomination_Assessment_Team). Tetapi ingatlah, jadwal mereka padat! Sadarlah akan banyaknya alternatif yang lebih cocok dibanding sesuatu yang kamu sarankan sebelumnya, jangan takut untuk mengesampingkan ego dirimu - **terkadang melakukan kesalahan bisa jauh lebih berharga ilmu dan pengalamannya dibanding menjadi benar setiap saat**.
```
Comment (mousesalt) [inline wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md:50]:
```suggestion
- Masih belum bisa mencapai kesepakatan? Minta opini kepada seseorang yang memiliki lebih banyak pengalaman, seperti anggota [BN](/wiki/People/The_Team/Beatmap_Nominators) atau [NAT](/wiki/People/The_Team/Nomination_Assessment_Team). Tetapi ingatlah, jadwal mereka padat! Sadarlah akan banyaknya alternatif yang lebih cocok dibanding sesuatu yang kamu sarankan sebelumnya, jangan takut untuk mengesampingkan ego dirimu - **terkadang kesalahan bisa jauh lebih berharga dibanding kebenaran.**.
```

Comment (Nivalyx) [reply wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md:50]: imo it's better to reword this to be like

`terkadang kesalahan dapat menjadi guru yang paling berharga.`

**Review by Nivalyx — APPROVED** [review]
looks alright for me @ZeroPyrozen @mousesalt

**Review by mousesalt — APPROVED** [review]

**Review by TicClick — APPROVED** [review]
